backend/app/tests/api/routes/test_login.py

```diff
@@ -79,6 +82,13 @@ def test_reset_password(
     assert r.status_code == 200
     assert r.json() == {"message": "Password updated successfully"}
 
+    # * db asserts
+    user_query = select(User).where(User.email == settings.FIRST_SUPERUSER)
+    user = db.exec(user_query).first()
+    db.refresh(user)
```

Comment:
```suggestion
```

I think the refresh is not necessary here.

Reply:
The problem here is not the cache, the problem here is some services make the commit but doesn't refresh the object since they are gonna response a message and not a model, in this way sometimes we need to refresh from test the object. In other cases when we do the refresh in the service because we are gonna return the model it's fine. So i remove them where they are not necessary and left which ones who are necessary or the test will fail

It's done now